src/Core/Content/Product/Helper/ProductVariationHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Product\Helper;
+
+use Shopware\Core\Content\Product\ProductEntity;
+use Shopware\Core\Content\Property\Aggregate\PropertyGroupOption\PropertyGroupOptionEntity;
+
+class ProductVariationHelper
```

Comment:
```suggestion
class ProductVariationBuilder
```

Put it into Product directory directly.